lib/maplibre-compose/src/commonMain/kotlin/org/maplibre/compose/location/LocationRequest.kt

```diff
@@ -0,0 +1,20 @@
+package org.maplibre.compose.location
+
+/**
+ * A request for a location update. This class is used to configure the location provider.
+ *
+ * @param position True if the position should be included in the location update, false otherwise.
+ *   Defaults to true.
+ * @param speed True if the speed should be included in the location update, false otherwise.
+ *   Defaults to true.
+ * @param course True if the course should be included in the location update, false otherwise.
+ *   Defaults to true.
+ * @param orientation True if the orientation should be included in the location update, false
+ *   otherwise. Defaults to false.
+ */
+public data class LocationRequest(
+  val position: Boolean = true,
+  val speed: Boolean = true,
+  val course: Boolean = true,
```

Comment:
Do these use separate APIs from `position` on any platform? If not, they are essentially free and there should be no reason to toggle them separately.